### PR TITLE
Added ctesque integration tests for Color class

### PR DESCRIPTION
**Pull request by mlebedy**
Tests added to check the pinning of saturation and value parameters in Color#HSVToColor.

### CODE REVIEW

File: integration_tests/ctesque/src/sharedTest/java/android/graphics/ColorTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.junit.runner.RunWith;
+import org.robolectric.annotation.internal.DoNotInstrument;
+
+/** Compatibility tests for {@link Color} */
+@DoNotInstrument
+@RunWith(AndroidJUnit4.class)
+public class ColorTest {
+  @Test
+  public void HSVToColorValueShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[] {0f, 0f, -1.0f})).isEqualTo(Color.BLACK);
```
Comment (utzcoz) [inline integration_tests/ctesque/src/sharedTest/java/android/graphics/ColorTest.java:16]: Maybe removing tests from shadows directory? This test file will be executed both in Robolectric environment and Emulator.

Comment (mlebedy) [reply integration_tests/ctesque/src/sharedTest/java/android/graphics/ColorTest.java:16]: Removed, thanks!

**Review by utzcoz — CHANGES_REQUESTED** [review]

File: robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java [hunk 2]
```diff
@@ -1,89 +0,0 @@
...
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-@RunWith(AndroidJUnit4.class)
-public class ShadowColorTest {
-
-  @Test
-  public void testRgb() {
-    int color = Color.rgb(160, 160, 160);
```
[the remark this answers is not in the document]
Comment (utzcoz) [reply robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java:15]: Or you can move all tests in `ShadowColorTest` to `ColorTest` as `ShadowColorTest` doens't use any shadow APIs.

[the remark this answers is not in the document]
Comment (utzcoz) [reply robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java:15]: I think they can work on Emulator too.

[the remark this answers is not in the document]
Comment (mlebedy) [reply robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java:15]: I am sorry, you are right, I totally forgot about the other tests in ShadowColorTest. I will try to move them all under ctestque and run the connectedTests with the emulator.

**Review by utzcoz — APPROVED** [review]
LGTM. Wait the CI.

**Pull request merged**